crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -528,6 +528,25 @@ impl<T: WasmInstance> WasmInstanceActor<T> {
     fn update_database(&mut self) -> Result<UpdateDatabaseResult, anyhow::Error> {
         let stdb = &*self.database_instance_context().relational_db;
 
+        // Until we know how to migrate schemas, we only accept `TableDef`s for
+        // existing tables which are equal sans their indexes.
+        struct Equiv<'a>(&'a TableDef);
```

Comment:
Kind of odd to have it in this function, but I also understand because of the fact that, it's really only meant to be used here, so I think it's fine.

Reply:
It's the most concise way of writing a somewhat janky `PartialEq` I could think of. Will also be gone when we have migrations.

Reply:
Well the pattern match will guard against not considering any additional fields, should this code outlive its intended lifespan. Imo inlining it would just further obscure the already rather complex logic.